NEWS

```diff
@@ -11,6 +11,88 @@
 ### Features:
 ### Bugfixes:
 
+## 1.19.1 (Sep 18, 2025)
```

Comment:
It seems we are missing 1.19.0's NEWS in the master branch too.

Reply:
@amastbaum  plz push 1.19.0 first, before this one